stdlib/public/core/Sort.swift

```diff
@@ -344,7 +344,7 @@ internal func _merge<Element>(
   high: UnsafeMutablePointer<Element>,
   buffer: UnsafeMutablePointer<Element>,
   by areInIncreasingOrder: (Element, Element) throws -> Bool
-) rethrows -> Bool {
```

Comment:
Although this is internal, since its `inlinable` this is part of stdlib ABI so not sure this can be changed